src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -208,6 +208,17 @@ protected Object decode(
                                 case 3:
                                     position.setDeviceTime(new Date(buf.readUnsignedIntLE() * 1000));
                                     break;
+                                case 4:
+                                    value = buf.readUnsignedByte();
+                                    position.set(Position.KEY_ALARM, BitUtil.check(value, 2));
```

Comment:
I just realized that this is incorrect. What alarm is this?

Reply:
This is a global alert status.  When the device is armed, the triggering of security sensors:
 1. Generates an Alarm event.
 2. Set the alarm flag (this bit).

 Further, all messages will be with the ALARM flag set.

 Reset bit only when disarming.

Reply:
Yeah, but the value cannot be boolean. It has to be one of the valid alarm types:

https://github.com/traccar/traccar/blob/master/src/main/java/org/traccar/model/Position.java#L114-L151

Reply:
Mmm..

This flag is more like the state the device has entered.  They can be lengthy.  
And the ALARM keys from your list are more like EVENT, or more precisely, the cause of the event.

Or I'm wrong?

Reply:
If it's in alarm state, you should send one of those for Traccar to handle it correctly.

Reply:
Ok, fix it